adapters/oidc/js/src/main/resources/keycloak.js

```diff
@@ -110,8 +110,9 @@
                 }
 
                 if (initOptions.promiseType === 'native') {
-                    kc.useNativePromise = typeof Promise === "function";
```

Comment:
I've removed the feature detection here as it doesn't make sense to fall back to the legacy API if the consumer of the API is expecting a different interface. Better to fail hard constructing a promise then to pretend we can fix things.